Code/Editor/Util/Platform/Mac/FileUtil_Mac.cpp

```diff
@@ -21,7 +21,7 @@ namespace Platform
         return QProcess::startDetached(command, argsList);
     }
 
-    bool RunEditorWithArg(const QString& editor, const QString& arg)
+    bool RunEditorWithArg(const QString editor, const QString arg)
     {
```

Comment:
The changes here are a Mac build fix in development, please revert this file to match development.

Reply:
incorporated more stabilization/2305 changes up to [that fix](https://github.com/o3de/o3de/commit/39dcbcf70c4a90632e83364cc757289086bf935d), should be ok now